mathics/builtin/numbers/constants.py

```diff
@@ -158,7 +159,7 @@ def to_mpmath(self, args):
         return getattr(mpmath, self.mpmath_name)
 
 
-class _NumpyConstant(_Constant_Common):
+class _NumpyConstant(_Constant_Common, NumericOperators):
```

Comment:
Don't know but will check and let you know...

Reply:
This `*Constant` are superclasses of some Builtins. Builtins are not `Element`s so, I think 
* They shouldn't be a subclass of `Number` that are Elements.
* In general, subclasses of `Builtin` shouldn't  implement `NumericOperators`.

Reply:
@mmatera, I don't understand this enough to give my opinion about the first, but I agree with the second.

Reply:
@mmatera Ok, it took me a while to understand what you mean, but yes, I believe you are correct and 1edd13d1 should address this. 

@TiagoCavalcante In effect what mmatera is saying is that inside Python you can't write say `Pi() + 3` (with or without the parenthesis) like you could do say `Integer(5) + 3`. This is because `Pi` acts as a built-in symbol which needs to be evaluated. You could presumably write in some code to do the evaluation, e.g. call the `evaluate()` function, but the the whole convenience of using infix `+` is kind of lost. 

So in summary, this kind of shorthand just doesn't apply for Mathics built-in variables/symbols like this.